src/playbook.yaml

```diff
@@ -2,11 +2,90 @@
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('devtools', repos=('https://cran.rstudio.com/'))\""
+
+    - name: Install synapser
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('synapser', repos=c('http://ran.synapse.org', 'https://cran.rstudio.com/'))\""
+
+    - name: Install tidyverse
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('tidyverse')\""
+
+    - name: Install shiny
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('shiny', repos='https://cran.rstudio.com/')\""
+
+    - name: Install BiocManager
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('biocManager', repos='https://cran.rstudio.com/')\""
+
+    - name: Install RStudio Professional Drivers
+      shell: "curl -O https://drivers.rstudio.org/7C152C12/installer/rstudio-drivers_1.6.0_amd64.deb && yes | gdebi rstudio-drivers_1.6.0_amd64.deb"
+
+    - name: Basic driver config
+      shell: "cp /etc/odbcinst.ini /etc/odbcinst.ini.bak && cat /opt/rstudio-drivers/odbcinst.ini.sample | tee /etc/odbcinst.ini"
+
+    # R 4.0.0
+    - name: Install R 4.0
+      shell: "curl -O https://cdn.rstudio.com/r/ubuntu-1804/pkgs/r-4.0.0_1_amd64.deb && yes | gdebi r-4.0.0_1_amd64.deb"
+
+    - name: Add R-Project apt repository
+      shell: "aptdcon --add-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu {{ ansible_distribution_release }}-cran40/' && aptdcon --refresh"
+
+    # Install R packages for R 4.0.0
+    - name: Install devtools
+      shell: "/opt/R/4.0.0/bin/R -e \"install.packages('devtools', repos=('https://cran.rstudio.com/'))\""
+
+    - name: Install synapser
+      shell: "/opt/R/4.0.0/bin/R -e \"install.packages('synapser', repos=c('http://ran.synapse.org', 'https://cran.rstudio.com/'))\""
+
+    - name: Install tidyverse
+      shell: "/opt/R/4.0.0/bin/R -e \"install.packages('tidyverse')\""
+
+    - name: Install shiny
+      shell: "/opt/R/4.0.0/bin/R -e \"install.packages('shiny', repos='https://cran.rstudio.com/')\""
+
+    - name: Install BiocManager
+      shell: "/opt/R/4.0.0/bin/R -e \"install.packages('biocManager', repos='https://cran.rstudio.com/')\""
+
+    - name: Install RStudio Professional Drivers
+      shell: "curl -O https://drivers.rstudio.org/7C152C12/installer/rstudio-drivers_1.6.0_amd64.deb && yes | gdebi  rstudio-drivers_1.6.0_amd64.deb"
+
+    - name: Basic driver config
+      shell: "cp /etc/odbcinst.ini /etc/odbcinst.ini.bak && cat /opt/rstudio-drivers/odbcinst.ini.sample | tee /etc/odbcinst.ini"
+
+    # Symlink R 4.0.0
+    - name: Symlink R 4.0.0
+      shell: "ln -s /opt/R/4.0.0/bin/R /usr/local/bin/R && ln -s /opt/R/4.0.0/bin/Rscript /usr/local/bin/Rscript"
+
+    # Install Shiny server Pro
+    - name: Download Shiny Server
+      shell: "wget https://s3.amazonaws.com/rstudio-shiny-server-pro-build/ubuntu-14.04/x86_64/shiny-server-commercial-1.5.13.1042-amd64.deb"
+
+    - name: Install Shiny Server
+      shell: "yes | gdebi shiny-server-commercial-1.5.13.1042-amd64.deb"
```

Comment:
Is installing the shiny server enough? It needs to be enabled right?

Reply:
Correct, I was just planning on adding the license key when it's running. We'll have to connect anyway to add dependencies that could be missing (or just apps), I just want to capture the basic (but time consuming) setup.